polkadot/network/src/lib.rs

```diff
@@ -363,6 +371,8 @@ impl PolkadotProtocol {
 				send_polkadot_message(ctx, peer_id, Message::BlockData(req_id, block_data));
 			}
 			Message::BlockData(req_id, data) => self.on_block_data(ctx, peer_id, req_id, data),
+			Message::Collation(relay_parent, collation) => self.on_collation(ctx, peer_id, relay_parent, collation),
+			Message::CollatorRole(_) => {},
```

Comment:
If I understand that Message right, this peer is informing us about our CollatorRole, no? And we just ignore it? Didn't it say up there in a TODO that we should get punished if we try to collate even if we aren't primary? Or is the implementation of this going to be part of the follow-up PR? Even in that case, maybe have a ToDo here, too?

Reply:
that's the follow-up PR. Can add a TODO but it'll follow-up really shortly so I don't know for whose benefit that is.